code/stable/dblpend/SRS/PDF/DblPend_SRS.tex

```diff
@@ -311,7 +322,7 @@ \subsubsection{Theoretical Models}
         
 \\ \midrule
 Equation & \begin{displaymath}
-           \symbf{a}\text{(}t\text{)}=\frac{\,d\symbf{v}\text{(}t\text{)}}{\,dt}
+           \resizeExpression{\symbf{a}\text{(}t\text{)}=\frac{\,d\symbf{v}\text{(}t\text{)}}{\,dt}}{0.8}
```

Comment:
We can predict that we'll likely want more processing of expressions in the future, so may as well abstract from that now. In other words, introduce a `\formatExpr` macro that wraps expressions, which would then just be `resizeExpression{#1}{0.8}`.

Reply:
### To clarify, are you suggesting the following?

In addition to the existing variables/commands I created in this PR, which are as follows:

```LaTeX
\def\inDefScale{0.8}
\def\outDefScale{1.0}

\newsavebox{\mybox}

\newcommand{\resizeExpression}[2]{
  \savebox{\mybox}{$#1$}
  \ifdim\wd\mybox>#2\linewidth
    \resizebox{#2\textwidth}{!}{\usebox{\mybox}}
  \else
    \usebox{\mybox}
  \fi
}
```

we want the following macro:

```LaTeX
\newcommand{\InDef}{true}
\newcommand{\OutDef}{false}

\newcommand{\formatExpr}[2]{
  \if#2
    \resizeExpression{#1}{\inDefScale}
  \else
    \resizeExpression{#1}{\outDefScale}
  \fi
}
```

Thus, expressions will be printed as follows:


```Latex
\formatExpr{...}{\outDef} % Outside definition table
\formatExpr{...}{\inDef} % Inside definition table
```

That’s how I understand it. Please let me know if I’ve misunderstood anything.